Networking/Poller.cc

```diff
@@ -16,26 +16,37 @@
 // limitations under the License.
 //
 
+#define NOMINMAX
```

Comment:
What's this for? Could you add a comment? And does it have to be at the very top of the file?0

Reply:
It stops the annoying min and max macros from being defined and clobbering std::min, etc.  I'll make that more clear.